rules/framework.bzl

```diff
@@ -273,6 +272,17 @@ def _apple_framework_packaging_impl(ctx):
         )
         _add_to_dict_if_present(objc_provider_fields, key, set)
 
+    for key in ["defines", "includes"]:
+        collected = []
+        for dep in ctx.attr.deps:
+            if CcInfo in dep:
+                collected.append(getattr(dep[CcInfo].compilation_context, key))
+        set = depset(
+            direct = [],
+            transitive = collected,
+        )
+        _add_to_dict_if_present(compilation_context_fields, key, set)
```

Comment:
`define` and `include` was removed from `ObjCProvider`. This block of code is based on the block above it that aggregates transitive attributes. I'm not sure if this part is necessary or if the merging of `CcInfo`s below will handle this?

Reply:
The tests seem to pass either way, unless there isn't anything in the tests exercising objc include dirs or defines.

Reply:
I think this will be handled by the merging of `CcInfo` below. We can just remove the two keys.